environment/deployments/data-curation/main.tf

```diff
@@ -115,7 +115,7 @@ module "storage_bucket_2" {
   }
 }
 
-// HIPS Storage Bucket
+// HIPS Storage Bucket (VISA testing)
```

Comment:
Was that "VISA" supposed to be "VISTA"?

Reply:
Yup, thanks for that and spotting the other typos. Fixed now.